index.js

```diff
@@ -237,6 +239,7 @@ type MaxValueValidationRule implements ValidationRule {
   custom: Int
   previousAnswer: BasicAnswer
   entityType: ValidationRuleEntityType
+  availablePreviousAnswers: [Answer]
```

Comment:
I think these can be not nullable so it would be `[Answer!]!` which can still return empty arrays but there is always an array and it is never "holey"